packages/esm-patient-test-results-app/src/index.ts

```diff
@@ -9,6 +9,13 @@ import { createDashboardLink } from '@openmrs/esm-patient-common-lib';
 import { configSchema } from './config-schema';
 import { dashboardMeta } from './dashboard.meta';
 
+export const hivMeta = {
+  name: 'Hiv-care-and-treatment',
+  slot: 'Hiv dashboard',
```

Comment:
```suggestion
  slot: 'hiv-dashboard-slot',
```

Reply:
convention to name slots with a `slot` suffix

Reply:
I've noticed that the kebab case syntax is common throughout the project. However I noticed in the Getting Started docs it says that the human readable name is preferred for Slot names? http://o3-dev.docs.openmrs.org/#/main/extensions?id=naming-slots Perhaps @brandones  can clarify

Reply:
Okay 👍🏽

Reply:
As a user of the implementer tools, I find it much easier to identify that something is a slot when it's name ends in "-slot". I believe that's been our convention. @brandones?

Reply:
Yeah I think I was foolish for me not to push against this harder. The obvious solution is that the implementer tools should just make it clearer what is a slot and what is not. If the best way to do this is to append "slot" when displaying the names, that should be done automatically by the tooling, not as a convention by humans who can and will screw it up.

Reply:
For the time being, until this change is made by the implementer tools, could we keep this convention (and get this PR merged)?

Reply:
Fine by me